Nathan-Chevalier/levelup-client
# Authorization - Login working

# Login successfully communicating with the API and setting token
## Steps to Achieve
- Aligned all sent values from the client to all expected values on the server (username, password)
- Configured `vite.config.js` to set port 3000 to align with server side expected port
## Steps to Test
- Ensure you have the latest version of main for `levelup-server`
- Pull down the `authorization` branch associated with this PR
- Start your API server in VS Code (Debugger)
- Navigate to `http://localhost:3000/` and log in with Username: `Carrie1945` Password: `me`
- You should be navigated to the placeholder "Home" page
